Tests/Unit/ParseSetupUnitTests.m

```diff
@@ -45,4 +45,22 @@ - (void)testInitializeWithLDSAfterInitializeShouldThrowException {
     [Parse setApplicationId:@"a" clientKey:@"b"];
     PFAssertThrowsInconsistencyException([Parse enableLocalDatastore]);
 }
+
+- (void)testInitializeWithNilApplicationIdNilClientKeyShouldThrowException {
+    NSString *yolo = nil;
+    PFAssertThrowsInconsistencyException([Parse setApplicationId:yolo clientKey:yolo]);
+}
+
+- (void)testInitializeWithNilApplicationIdShouldThrowException {
+    NSString *yolo = nil;
+    PFAssertThrowsInconsistencyException([Parse setApplicationId:yolo clientKey:@"a"]);
```

Comment:
If you want to avoid the temporary variable, you can cast `nil` to an `id` to suppress the warning IIRC.